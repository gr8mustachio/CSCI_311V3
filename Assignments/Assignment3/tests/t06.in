recFind
5 13
24 79 38 13 62
recFind
6 17
12 23 14 75 39 42
recFind
41 1791538 
1782348	1642092	1600371	1738340	1929421	1844963	1799238	1655674	1960573	1977884	1698734	1833214	1797367	1908133	1611574	1749643	1968567	1817722	1790538	1927514	1797367	1927614	1790538	1817722	1968567	1749643	1611574	1908133	1797367	1833214	1698734	1977884	1960573	1655674	1799238	1844963	1929421	1738340	1600371	1642092	1782348
recFind 
40 1799238
1782348 1642092 1600371 1738340 1929421 1844963 1799238 1655674 1960573 1977884 1698734 1833214 1797367 1908133 1611574 1749643 1968567 1817722 1790538 1927514 1797367 1927614 1790538 1817722 1968567 1749643 1611574 1908133 1797367 1833214 1698734 1977884 1960573 1655674 1799238 1844963 1929421 1738340 1600371 1642092
