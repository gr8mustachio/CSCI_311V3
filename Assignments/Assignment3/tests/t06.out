Array A:
24 79 38 13 62 
13 is found in unordered A.
Array A:
12 23 14 75 39 42 
17 is NOT found in unordered A.
Array A:
1782348 1642092 1600371 1738340 1929421 1844963 1799238 1655674 1960573 1977884 1698734 1833214 1797367 1908133 1611574 1749643 1968567 1817722 1790538 1927514 1797367 1927614 1790538 1817722 1968567 1749643 1611574 1908133 1797367 1833214 1698734 1977884 1960573 1655674 1799238 1844963 1929421 1738340 1600371 1642092 1782348 
1791538 is NOT found in unordered A.
Array A:
1782348 1642092 1600371 1738340 1929421 1844963 1799238 1655674 1960573 1977884 1698734 1833214 1797367 1908133 1611574 1749643 1968567 1817722 1790538 1927514 1797367 1927614 1790538 1817722 1968567 1749643 1611574 1908133 1797367 1833214 1698734 1977884 1960573 1655674 1799238 1844963 1929421 1738340 1600371 1642092 
1799238 is found in unordered A.
